src/components/Breadcrumbs/component.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useMemo } from 'react';
+
+import { Testable, useBuildTestId } from '../../modules/test-ids';
+
+import { Context } from './context';
+import { Context as ItemContext } from './Item/context';
+import { Styled } from './styled';
+
+export type Props = Testable & {
+  children: React.ReactNode;
+  className?: string;
+};
+
+export const Component = ({ 'data-testid': testId, ...otherProps }: Props) => {
+  const { buildTestId } = useBuildTestId({ id: testId });
+  const context = useMemo(() => ({ testId }), [testId]);
+
+  const breadcrumbs = useMemo(() => {
+    const children = React.Children.toArray(otherProps.children);
+
+    return children.map((it, index) => {
+      return (
```

Comment:
Optional I think, a shorthand `() => (<ItemContext.Provider>...</ItemContext.Provider>)` would be enough

Reply:
Done!